src/main/java/tuteez/model/person/lesson/Day.java

```diff
@@ -1,33 +1,62 @@
 package tuteez.model.person.lesson;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * Enum to represent the days of the week.
  */
 public enum Day {
-    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY;
+    MONDAY("mon"),
+    TUESDAY("tue"),
+    WEDNESDAY("wed"),
+    THURSDAY("thu"),
+    FRIDAY("fri"),
+    SATURDAY("sat"),
```

Comment:
Interesting implementation, I did not know this could be done. This looks good too!! I'll look more into it when i have the chance. Good work